tools/bench.ml

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/ocaml
```

Comment:
Cool!

Reply:
well in fact I have to remove that script, it is kind of useless